app/pb_kits/playbook/data/menu.yml

```diff
@@ -7,6 +7,8 @@ samples:
 kits:
   - avatar
   - avatar_action_button
+  - background
+  - badge
```

Comment:
So is this a bug fix? Was badge not listed on the main page?

Reply:
i think it was a dupe